src/particles/collisions/ElasticCollisionPerez.H

```diff
@@ -153,15 +153,15 @@ void ElasticCollisionPerez (
             // particle's Lorentz factor
             amrex::Real g1 = is_beam_coll ? std::sqrt( 1._rt
                 + u1x[I1[i1]]*u1x[I1[i1]] + u1y[I1[i1]]*u1y[I1[i1]] + psi1[I1[i1]]*psi1[I1[i1]] )
-                : (1.0_rt + u1x[I1[i1]]*u1x[I1[i1]] + u1y[I1[i1]]*u1y[I1[i1]] +
-                   psi1[I1[i1]]*psi1[I1[i1]]) / (2.0_rt * psi1[I1[i1]] );
+                : plasma_gamma(u1x[I1[i1]], u1y[I1[i1]], psi1[I1[i1]],
+                    1._rt / psi1[I1[i1]], /* Assumes Aabssq == 0 */ 0._rt);
```

Comment:
Difficult to say... First-order, I think what you write is appropriate for electron-electron collision, less for electron-ion. Let's keep like this.

Reply:
Maybe we can fix it later if we need it. It would require an extra laser gathering step.